db/migrate/20190812101027_create_applications.rb

```diff
@@ -1,10 +1,11 @@
-class CreateApplication < ActiveRecord::Migration[5.2]
+class CreateApplications < ActiveRecord::Migration[5.2]
   def change
     create_table :applications do |t|
```

Comment:
```suggestion
    create_table :apps do |t|
```
the same for model

Reply:
let's keep name of tables full-named. I mean, let's use no reductions